src/components/ErrorPage/ErrorPage.js

```diff
@@ -0,0 +1,144 @@
+		fontSize: theme.typography.fontSize * 1.5,
+		lineHeight: 1.76,
+		textAlign: 'center',
+		marginTop: theme.spacing(7),
+		marginBottom: theme.spacing(4),
+		marginLeft: theme.spacing(2.75),
+		marginRight: theme.spacing(2.75),
+		[theme.breakpoints.down('xs')]: {
+			fontSize: theme.typography.fontSize,
+			lineHeight: 1.57,
+			marginTop: theme.spacing(3),
+			marginBottom: theme.spacing(3)
+		}
+	},
+	pageContainer: { // TODO: add padding on the left and right -> OPTIMIZE FOR MOBILE
+		minHeight: '100vh'
+	},
+	hiddenImage: {
+		textAlign: 'center',
+		marginTop: theme.spacing(4),
+		placeItems: 'center'
+	},
+	hackheart: {
+		width: '95%'
+	}
+});
+
+function ErrorPage({ classes }) {
+	const data = useStaticQuery(graphql`
+	query {
+	  bowimg: file(relativePath: {eq: "404/bowimg-removebg.png"}) {
+	    childImageSharp {
+	      fixed(height: 400) {
+	          base64
+	          width
+	          height
+	          src
+	          srcSet
+	      }
+	    }
+		}
+		pixelheart: file(relativePath: {eq: "404/pixelheart.png"}) {
+	    childImageSharp {
+	      fixed(height: 22) {
+	          base64
+	          width
+	          height
+	          src
+	          srcSet
+	      }
+	    }
+		}
+		hackheart: file(relativePath: {eq: "404/hackheart.png"}) {
+	    childImageSharp {
+	      fluid {
+	        base64
+          aspectRatio
+          src
+          srcSet
+          sizes
+        }
+      }
+    }
+  }`);
+	const [imageOpen, setImageOpen] = useState(false);
+	return (
+		<Container maxWidth="md" className={classes.pageContainer}>
+
+			<div className={classes.container}>
+				<div><Typography variant='h3' >NOT FOUND</Typography></div>
+				<p>
+					<Typography className={classes.content}>
+						Oops! This page has either been taken down or does not exist
+					</Typography>
+				</p>
+				<div><Img fixed={data.bowimg.childImageSharp.fixed} alt='bowimg' /></div>
+			</div>
+			<div className={classes.container}>
+				<Typography display='inline' variant='h5'>
+					With love {' '}
+					<div style={{ display: 'inline' }}
+						onClick={() => setImageOpen(!imageOpen)}>
+						<Img fixed={data.pixelheart.childImageSharp.fixed}
+							alt='404 Meme'
+						/>
+					</div>
+					{' '}from Hack
+				</Typography>
+			</div>
+			<div
+				className={classes.hiddenImage}
+				style={{
+					display: imageOpen ? 'grid' : 'none'
+				}}
+				id='hackheart'
+			>
+				<Img className={classes.hackheart} fluid={data.hackheart.childImageSharp.fluid} alt='hackheart' />
+			</div>
+			<div className={classes.container}>
+				<Link
+					href='https://hack.uclaacm.com/'
+					color='textSecondary'
+					underline='none'
+				>
+					<Typography>Go back to Home</Typography>
+				</Link>
+				<Link
+					href='https://github.com/uclaacm/hackschool-f19/tree/master/session-3-backend-api#http-responses'
+					color='textSecondary'
+					underline='none'
+				>
+					<Typography>Why am I seeing this message?</Typography>
```

Comment:
We might wanna specify `component="span"`. Currently, the link has the tag `<a><p>...</p></a>`. We can use `span` since it used for decorating text which is already in a paragraph or a link.